app/views/settings/index.html.erb

```diff
@@ -15,6 +15,10 @@
             <%= f.label :email %>
             <%= f.email_field :email, class: "form-control input-lg" %>
           </div>
+          <div class="form-group">
```

Comment:
This is *one* way of inputting the UID, but we want the user to get their uid using oauth.